cpp/benchmarks/common/generate_benchmark_input.cpp

```diff
@@ -31,6 +31,7 @@
 
 #include <future>
 #include <memory>
+#include <optional>
```

Comment:
Thanks for finding this benchmark issue, I have updated #10044 to address this as well.

Reply:
Reverted.